torch/csrc/distributed/rpc/script_functions.cpp

```diff
@@ -50,6 +50,30 @@ c10::intrusive_ptr<c10::ivalue::Future> rpcTorchscriptCall(
   return futPtr;
 }
 
+PyRRef remoteTorchscript(
```

Comment:
could we refactor pyRemoteBuiltin and make it to be used by both builtin and torchscript call?

Reply:
I tried, the inheritance setup about ScripCall, RemoteCall, ScriptCall makes it hard to consolidate in this PR. It's could be another PR's work to streamline them.